.pylint.rc

```diff
@@ -26,7 +26,7 @@ logging-modules=logging,structlog
 [MESSAGES CONTROL]
 
 disable=all
-enable=no-value-for-parameter,too-many-format-args,no-member,bad-except-order,redefined-builtin,bad-continuation,unused-variable,no-self-use,import-self,gevent-joinall,useless-object-inheritance,unused-argument,unexpected-keyword-arg
+enable=no-value-for-parameter,too-many-format-args,no-member,bad-except-order,redefined-builtin,bad-continuation,unused-variable,no-self-use,import-self,gevent-joinall,useless-object-inheritance,unused-argument,unexpected-keyword-arg,expression-not-assigned,pointless-statement,unused-import,inconsistent-return-statements,reimported
```

Comment:
I steal these into `raiden-contracts`.  https://github.com/raiden-network/raiden-contracts/issues/816

Reply:
@pirapira if you can, please just enable all the rules

Reply:
I tried once all the rules, and backed off.